forge-gui/res/editions/Battlebond.txt

```diff
@@ -262,9 +262,81 @@ ScryfallCode=BBD
 252 L Swamp @Titus Lunter
 253 L Mountain @Titus Lunter
 254 L Forest @Titus Lunter
+
+[etched]
 255 M Will Kenrith @Anna Steinbauer
 256 M Rowan Kenrith @Anna Steinbauer
 
+[RareMythic]
```

Comment:
I don't really like the idea of using RareMythic as a name here, since it's a keyword for editions. (Which is probably why the previous version was using it incorrectly)

Reply:
Ok. renamed.